### ISSUE TITLE
Investigate into learning fate/entropy decay

### ISSUE THREAD

**Issue by deaconsham** [deaconsham/rl-bobble-league-agent#16]
The model can sometimes suck, where the agent finds a boring strategy and stops trying new things before it ever discovers how to score.

- [ ] Add an entropy coefficient to the PPO loss function in `training_loop.py` to reward the agent for being "random" and exploring.
- [ ] Use a learning rate scheduler to slowly lower the learning rate as the model gets smarter.
